apps/web-giddh/src/app/vat-report/vatReport.component.html

```diff
@@ -1,49 +1,19 @@
 <section appTranslate [file]="'vat-report'" (localeData)="localeData = $event"
     (commonLocaleData)="commonLocaleData = $event">
-    <nav class="navbar">
-        <div class="relative w-100">
-            <hamburger-menu [pageHeading]="localeData?.tab_heading"></hamburger-menu>
-            <div class="invoice-control w-100">
-                <div class="row mr-0 ml-0">
-                    <div class="vat-report w-100" id="setting-tab">
-                        <tabset #staticTabs>
-                            <tab heading="{{commonLocaleData?.tab_heading}}">
-                                <div *ngIf="currentCompanyBranches?.length > 2 && currentOrganizationType === 'COMPANY'"
-                                    class="row mb-3 mt-3">
-                                    <div class="tax-heading col-md-3 col-sm-3 col-12">
-                                        <label>{{commonLocaleData?.entity}}:</label>
-                                    </div>
-                                    <div class="tax-dropdown col-md-9 col-sm-9 col-12 linked-entities">
```

Comment:
This code getting removed. Need to revert

Reply:
done